op-node/rollup/sync/start.go

```diff
@@ -156,8 +156,14 @@ func FindL2Heads(ctx context.Context, cfg *rollup.Config, l1 L1Chain, l2 L2Chain
 		if (n.Number == result.Finalized.Number) && (n.Hash != result.Finalized.Hash) {
 			return nil, fmt.Errorf("%w: finalized %s, got: %s", ReorgFinalizedErr, result.Finalized, n)
 		}
+
+		// If the config didn't set MaxReorgSeqWindows, use the default.
+		maxReorgSeqWindows := uint64(MaxReorgSeqWindows)
```

Comment:
I think we should renamve `MaxReorgSeqWindows` to `DefaultMaxReorgSeqWindows` and during config parsing, set the value on the config struct to the default if its 0 or missing in the config file

Reply:
Then we can default to using the config struct in this codepath no matter what

Reply:
Yes can do. Is there a single path for this? Just the json load from file method? I was worried I wouldn't cover all of them (if there are multiple)